tests/dummy/app/controllers/select.js

```diff
@@ -1,23 +1,21 @@
 import Controller from '@ember/controller';
-import { get, set } from '@ember/object';
 import { A } from '@ember/array';
+import { tracked } from '@glimmer/tracking';
 
-export default Controller.extend({
-    selectLgValue: null,
-    inlineSelectOptions: A([
+export default class SelectController extends Controller {
+    @tracked selectLgValue;
+    @tracked selectInlineValue;
+    @tracked selectBlockValue;
+
+    inlineSelectOptions = A([
         {value: "a", label: "Apples"},
         {value: "b", label: "Bananas"},
         {value: "c", label: "Oranges"}
-    ]),
+    ]);
 
-    customSelectOptions: A([
+    customSelectOptions = A([
         {abbrv: "NY", name: "New York"},
         {abbrv: "NJ", name: "New Jersey"},
         {abbrv: "CT", name: "Connecticut"}
-    ]),
-
-    init() {
-        this._super(...arguments);
-        set(this, 'selectLgValue', get(this, 'customSelectOptions.firstObject'));
-    }
-});
+    ]);
+}
```

Comment:
we don't need these dummy files to be typescript?

Reply:
nah, and might even be better off staying as js in some respects, as we want to make sure the addon works with non-TS apps too, as other people in the community might use it.